Tests/SnapshotTests/Components/UPIComponentUITests.swift

```diff
@@ -145,29 +156,30 @@ class UPIComponentUITests: XCTestCase {
             XCTAssertEqual(data.type, "upi_intent")
             didSubmitExpectation.fulfill()
         }
-
-        wait(for: .milliseconds(300))
         
-        sut.currentSelectedIndex = 0
-        sut.currentSelectedItem = sut.upiAppsList.first
-        sut.currentSelectedItem?.isSelected = true
-        sut.virtualPaymentAddressItem.isHidden.wrappedValue = true
+        sut.upiAppsList.first?.selectionHandler?()
 
+        // TODO: Wait until the first item is selected
```

Comment:
Forgotten or gonna be followed up in a second PR?

Reply:
Yesss forgotten - removed it 👌